recipes/unassigner/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "unassigner" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0a79a0fdf88318a2583e2a46deffb6f6ff4ce612c55b470c007eed7bb9aa2194
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - unassigner = unassigner.command:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  run_exports:
+    - {{ pin_subpackage('unassigner', max_pin="x") }}
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - vsearch
+
+test:
+  imports:
+    - unassigner
+  commands:
+    - unassigner --help
+
+about:
+  home: https://github.com/PennChopMicrobiomeProgram/unassigner
+  license: MIT
+  license_family: MIT

```

Comment:
It's `GPLv2` in https://github.com/PennChopMicrobiomeProgram/unassigner/blob/23ddca1c270d8a31067229d0f912aafbed8e6d75/setup.py#L39
is it possible to add a LICENSE file to the original repo as well ?